server/pulp/server/event/http.py

```diff
@@ -24,7 +26,7 @@
 def handle_event(notifier_config, event):
     # fire the actual http push function off in a separate thread to keep
     # pulp from blocking or deadlocking due to the tasking subsystem
-    data = event.data()
+    data = json.dumps(event.data(), default=json_util.default)
```

Comment:
I'll need to test this (tomorrow), but I _think_ you'll need to change the kwarg used in the `post` call in `_send_post` to be `data` rather than `json`[0]. It may be that requests just does the right thing, though.

[0] http://docs.python-requests.org/en/master/api/#requests.post